draft-ietf-quic-http.md

```diff
@@ -1056,9 +1055,9 @@ byte stream to the application. Although QUIC permits out-of-order delivery
 within a stream, HTTP/3 does not make use of this feature.
 
 QUIC streams can be either unidirectional, carrying data only from initiator to
-receiver, or bidirectional.  Streams can be initiated by either the client or
-the server.  For more detail on QUIC streams, see
-{{Section 2 of QUIC-TRANSPORT}}.
+receiver, or bidirectional, carrying data in both directions.  Streams can be
+initiated by either the client or the server.  For more detail on QUIC streams,
+see {{Section 2 of QUIC-TRANSPORT}}.
```

Comment:
RFC Editor asked:
> Should this sentence be made parallel by explaining both unidirectional and bidirectional?  Or may we remove the explanatory clause as unidirectional is used several times prior to this occurrence?

I opted to parallelize, since I don't see the initiator-to-receiver direction stated elsewhere, but I'm also happy to shorten the text if it's adequately covered elsewhere.